### PR TITLE
AIRNEIS-112 : Frontend HTTP API + Config bugfixes

### PR DESCRIPTION
**Pull request by BarnabePILLIAUDIN**
# DONE 

- Created a request functions 
- Created a function per HTTP method
- Created a function to get,post,delete,or update category
- Made a test endpoint to be sure those functions works

# TEST

- Create a .env.local file into your front folder
- Copy the content of .env.local.exemple into it 
- MAKE SURE YOU DON'T HAVE A CATEGORY WITH THE ID 127 
- Make a get request to this endpoint `localhost:3000/api/test/categoriesFunctions`

### CODE REVIEW

**Review by cedricgautier — APPROVED** [review]

**Comment by hallycsse** [comment]
LGTM

**Review by hallycsse — APPROVED** [review]

File: front/src/pages/api/test/categoriesFunctions.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  ID: 127,
+  name: "Test",
+  thumbnailUrl: "https://picsum.photos/id/48/200/300",
+  slug: "test",
+  orderOfDisplay: 127,
+  description:
+    "Imperdiet veniam aliquip gubergren illum incidunt nonumy nonummy clita doming consetetur. Gubergren sanctus reprehenderit culpa laborum iure amet iriure labore accusam invidunt. Veniam mazim fugiat. Amet vulputate magna."
+}
+
+// BEFORE STARTING THE TEST BE SURE THAT YOU DO NOT HAVE A CATEGORY THAT HAVE 127 AS ORDER OF DISPLAY OR 127 AS ID!!
```
Comment (avetisk) [inline front/src/pages/api/test/categoriesFunctions.js:14]: `s/HAVE/HAS/g`.

File: front/src/utils/services/CategoryServices.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { apiRoutes } from "@/utils/constants"
+import { del, get, patch, post } from "@/utils/httpClient"
+
+export const getCategories = async () => await get(apiRoutes.categories)
+export const getCategoryById = async id => await get(apiRoutes.categoryId(id))
+export const createCategory = async (category, token) => await post(apiRoutes.categories, token, category)
+export const deleteCategory = async (id, token) => await del(apiRoutes.categoryId(id), token)
```
Comment (avetisk) [inline front/src/utils/services/CategoryServices.js:7]: Useless `await`.

Comment (avetisk) [reply front/src/utils/services/CategoryServices.js:7]: You can safely remove all the `async/await` in this file.

File: front/src/pages/api/test/categoriesFunctions.js (hunk 1 above)
Comment (cedricgautier) [reply front/src/pages/api/test/categoriesFunctions.js:14]: @BarnabePILLIAUDIN, fyi check SED : https://www.gnu.org/software/sed/manual/sed.html

**Pull request merged**